python/tests/test_account.py

```diff
@@ -2552,7 +2550,9 @@ def test_delete_multiple_messages(self, acfactory, lp):
 
         lp.sec("imap2: test that only one message is left")
         imap2 = ac2.direct_imap
-
+        imap2.idle_start()
+        imap2.idle_wait_for_new_message(timeout=600)
```

Comment:
Same as above, why 10 minutes here?

Reply:
idle was wrong to use here to begin with, i removed it.